.prettierrc.yaml

```diff
@@ -0,0 +1,2 @@
+singleQuote: true

```

Comment:
Similar to _.eslintrc.js_, this file exports the main Prettier configuration. Here's the [documentation](https://prettier.io/docs/en/options.html) for the various Prettier options. I've overridden `singleQuote` and `trailingComma` because their default values didn't fit this codebase